Test/test.cpp

```diff
@@ -0,0 +1,118 @@
+#include "pch.h"
+#include "../casbin/enforcer.h"
+#include "../casbin/rbac/default-role-manager/default_role_manager.h"
+
+TEST(EnforcerTest, MoreParameters) {
+	Error err;
+	Enforcer e = Enforcer("..\\..\\casbin\\examples\\MoreParam.conf", "..\\..\\casbin\\examples\\MoreParam.csv");

```

Comment:
path not handled correctly which makes test failed.

Reply:
I run it success at local.I will check what is happening.

Reply:
I am using Windows.Maybe you are using Linux,and the "\\" is not supported in Linux.
I will use '/' instead of '\\'